build/llhttp.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { join, resolve } = require('path')
+const { copyFileSync, rmSync } = require('fs')
+const { execSync } = require('child_process')
+const TMP = require('os').tmpdir()
+
+const REPO = 'git@github.com:dnlup/llhttp.git'
+const CHECKOUT = 'undici_wasm'
+const REPO_PATH = join(TMP, 'llhttp')
```

Comment:
Please make sure to use a tag or commit before landing this.

Reply:
Yes I will

Reply:
One thing though, do we want to use my personal repo? Or do we want to move that to another repo?

Reply:
preferably it should be part of llhttp repo? I think I need to defer to @mcollina here. I'm fine with pointing to your repo as long as its a tag/commit.